### PR TITLE
Pretreat 2015

### PR DESCRIPTION
**Pull request by dohmatob**
- removing zombie scripts
- refactoring examples with redundant code
- now full-dependent on nilearn's  fetching API
- updating examples to reflect above changes
- more.

### CODE REVIEW

File: examples/spm_multimodal_anat_only_preproc.ini [hunk 1]
```diff
@@ -1,186 +0,0 @@
...
-
-# Root directory (full path or relative to the directory containing this file) for
-# all output files and reports
-output_dir = ./pypreprocess_output
-
-# Generate html reports ?
-report = True
-
-# Plot coefficient of variation post-preprocessing ?
-plot_cv_tv = True
```
Comment (bthirion) [inline examples/spm_multimodal_anat_only_preproc.ini]: Just to understand: why do you remove that one ?

File: examples/unicog_struct.ini [hunk 2]
```diff
@@ -1,192 +0,0 @@
...
-
-# Root directory (full path or relative to the directory containing this file) for
-# all output files and reports
-output_dir = /neurospin/tmp/pypreprocess_output
-
-# Generate html reports ?
-report = True
-
-# Plot coefficient of variation post-preprocessing ?
-plot_cv_tv = True
```
Comment (bthirion) [inline examples/unicog_struct.ini]: Same question here.

File: examples/nipy_glm_spm_auditory.py [hunk 3]
```diff
@@ -1,197 +0,0 @@
...
-    n_scans=n_scans,
-    hfcut=hfcut,
-    frametimes=frametimes,
-    drift_model=drift_model,
-    hrf_model=hrf_model,
-    )
-
-base_reporter.ProgressReport().finish_dir(OUTPUT_DIR)
-
-print "\r\nStatistic report written to %s\r\n" % stats_report_filename
```
Comment (bthirion) [inline examples/nipy_glm_spm_auditory.py]: And that one ?

**Comment by bthirion** [comment]
LGTM. Just make clear why you remove the examples/scripts.

File: examples/spm_multimodal_anat_only_preproc.ini (hunk 1 above)
Comment (dohmatob) [inline examples/spm_multimodal_anat_only_preproc.ini]: This .ini had become obsolete and irrelevant.

File: examples/nipy_glm_spm_auditory.py (hunk 3 above)
Comment (dohmatob) [inline examples/nipy_glm_spm_auditory.py]: Was redundant with examples/nipype_*_auditory*.py

**Comment by dohmatob** [comment]
OK, merging.

**Pull request merged**